app/Resources/CraueConfigBundle/translations/CraueConfigBundle.pt.yml

```diff
@@ -11,9 +11,11 @@ export_txt: Habilita exportação para TXT
 export_xml: Habilita exportação para XML
 # import_with_rabbitmq: Enable RabbitMQ to import data asynchronously
 # import_with_redis: Enable Redis to import data asynchronously
-shaarli_url: URL Shaarli, se o serviço está habilitado
-scuttle_url: URL Scuttle, se o serviço está habilitado
-unmark_url: URL Unmark, se o serviço está habilitado
+shaarli_url: URL Shaarli caso o serviço esteja habilitado
+scuttle_url: URL Scuttle caso o serviço esteja habilitado
+unmark_url: URL Unmark caso o serviço esteja habilitado
+pocket_consumer_key: Chave de consumidor do Pocket para importar conteúdo (https://getpocket.com/developer/docs/authentication)
+shaarli_url: URL Shaarli caso o serviço esteja habilitado
```

Comment:
That line is duplicated
https://travis-ci.org/wallabag/wallabag/jobs/317631332#L1182